packages/cli/src/commands/build.ts

```diff
@@ -278,7 +278,7 @@ export async function build({
 
     if (persist) {
       await resolver.publish(
-        [`${name}:latest`, `${name}:${version}@${selectedPreset}`],
```

Comment:
This line caused a bug where the preset is being added to the filename in the tags directory, causing the scanDeploy / readDeploy functions to be unable to find the package deploy data